doc/advanced-topics.md

```diff
@@ -306,7 +306,8 @@ stored in the Context Broker by adding a new attribute to the entity with the sa
 element. By adding the field `explicitAttrs` with `true` value to device or group provision, the IoTAgent rejects the
 measure elements that are not defined in the mappings of device or group provision, persisting only the one defined in
 the mappings of the provision. If `explicitAttrs` is provided both at device and group level, the device level takes
-precedence.
+precedence. Additionally `explicitAttrs` can be used to define which meassures defined in JSON/JEXL array will be
+propagated to NGSI interface.
```

Comment:
The four cases described at https://github.com/telefonicaid/iotagent-node-lib/issues/1179#issuecomment-1014355699 should be detailed here.

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1180/commits/ed09484acfaf478de6e6ec1ef2a7fa1372e99a95